sources/commands/option_external.c

```diff
@@ -28,14 +33,33 @@ static char	*find_executable(char *tgt, t_dlist *path)
 	return (0);
 }
 
-static int	run_executable(char *fullpath, t_param *param)
+static int	run_executable(char *fullpath, t_treenode *node, t_param *param)
```

Comment:
for mock_8 node would be 
./tests/tool_print_environment 
its children:
    one 
    "two   three" 
    four